src/crate/client/sqlalchemy/doctests/itests.txt

```diff
@@ -80,9 +80,6 @@ Date should have been set at the insert due to default value via python method::
     >>> dt.day == now.day
     True
 
-    >>> (now - location.datetime_tz).seconds < 4
-    True
-
```

Comment:
Did this test case protect against anything significant? I can not make any sense of it.